### PR TITLE
Add _candidates_en/SotaroKimura_1.md

### LINKED ISSUE #104: 英訳: 木村宗太郎 さんによる「ローカル実行からお手軽に始めるSpark Structured Streaming」
ref: #100

### PR DESCRIPTION
**Pull request by richardimaoka**
Closes #104

### CODE REVIEW

File: _candidates_en/SotaroKimura_1.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Sotaro Kimura
+title: "Start playing with Spark Structured Streaming locally and build up a stream processing pipeline"
+length: 40
+audience: Beginner
+language: Japanese
+twitter: kimutansk
+github: kimutansk
+icon: https://s.gravatar.com/avatar/5d869b93035410ece41ff0d86d8c5c14
+organization: DWANGO Co., Ltd.
```
Comment (richardimaoka) [inline _candidates_en/SotaroKimura_1.md]: Confirmed the company name here http://dwango.co.jp/english/information/.

File: _candidates_en/SotaroKimura_1.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+tags:
+  - Big Data / Fast Data
+  - Tools
+suggestions:
+  - People who are interested in Spark and want to learn basics
+  - People who want to know practical use cases of Spark
+  - People who want to build a stream processing system quickly
+---
+Spark is a framework for parallel stream processing, however, that probably sounds like something difficult to work with even if you are interested.
```
Comment (richardimaoka) [inline _candidates_en/SotaroKimura_1.md]: work with or work on?

File: _candidates_en/SotaroKimura_1.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+Actually, Spark is a data processing tool which allows you execute it on your local machine as well as a large-scale cluster.
+At the same time, Scala users will feel familiar with it as it is OSS in Scala.
+Thus, you can start with your local machine quickly to run a data processing pipeline in Scala and keep building up on it.
```
Comment (richardimaoka) [inline _candidates_en/SotaroKimura_1.md]: start with or start on?

File: _candidates_en/SotaroKimura_1.md (hunk 2 above)
Comment (smroy) [reply _candidates_en/SotaroKimura_1.md]: either can be used, but work with is appropriate in this case.

File: _candidates_en/SotaroKimura_1.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+tags:
+  - Big Data / Fast Data
+  - Tools
+suggestions:
+  - People who are interested in Spark and want to learn basics
+  - People who want to know practical use cases of Spark
+  - People who want to build a stream processing system quickly
+---
+Spark is a framework for parallel stream processing, however, that probably sounds like something difficult to work with even if you are interested.
+Actually, Spark is a data processing tool which allows you execute it on your local machine as well as a large-scale cluster.
```
Comment (smroy) [inline _candidates_en/SotaroKimura_1.md]: Consider changing to "+In reality, Spark is a data processing tool which allows you to execute it on your local machine as well as in a large-scale cluster."

File: _candidates_en/SotaroKimura_1.md (hunk 3 above)
Comment (smroy) [inline _candidates_en/SotaroKimura_1.md]: Consider "+Thus, you can start it quickly with your local machine to run a data processing pipeline in Scala and then keep building upon it."

File: _candidates_en/SotaroKimura_1.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+At the same time, Scala users will feel familiar with it as it is OSS in Scala.
+Thus, you can start with your local machine quickly to run a data processing pipeline in Scala and keep building up on it.
+Moreover, processing structured data types like string and numeral types, as well as stream processing are both possible.
+This session will introduce how to build up a stream processing pipeline easily using Spark's new feature, Spark Structured Streaming,
+which allows you concisely write stream data processing as structured data.
```
Comment (smroy) [inline _candidates_en/SotaroKimura_1.md]: Change to "+,which allows you to concisely write stream data processing as structured data."

**Comment by smroy** [comment]
@richardimaoka I have added suggested changes.

**Comment by richardimaoka** [comment]
Thanks @smroy. I reflected your comments, could you approve this now?

**Pull request merged**